src/frontend/src/components/common/Collapsible/index.tsx

```diff
@@ -1,14 +1,27 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import CollapsibleProps from './interfaces';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import {
   faChevronLeft,
   faChevronRight,
 } from '@fortawesome/free-solid-svg-icons';
 
-const Collapsible = ({ icon, alignment, children }: CollapsibleProps) => {
-  const [isOpen, setIsOpen] = useState<boolean>(false);
+const Collapsible = ({
+  icon,
+  alignment,
+  children,
+  isRightOpen,
```

Comment:
I think that we may want to handle the right vs left sidebar open logic in the parent component (PIAForm)